api/_globals/structuredClone.json

```diff
@@ -61,7 +61,7 @@
           }
         },
         "status": {
-          "experimental": true,
+          "experimental": false,
```

Comment:
NOTE, changed this to false now is on 2 platforms - OK? 

BUT , I did NOT change the worker support setting experimental setting though, because deno does not support this in workers according to existing setup, untested). Feels wrong.

Reply:
Worker support in Deno is `true`. Let's set 'em both to `experimental: false`

Reply:
Dunno how I missed that. Thanks. Fixed.